routers/api/packages/container/container.go

```diff
@@ -448,9 +448,9 @@ func PutBlobsUpload(ctx *context.Context) {
 		return
 	}
 
-	// There was a strange bug: the "Close" fails with error "close .../tmp/package-upload/....: file already closed"
-	// AFAIK there should be no other "Close" call to the uploader between NewBlobUploader and this line.
-	// At least it's safe to call Close twice, so ignore the error.
+	// Some SDK (e.g.: minio) will close the Reader if it is also a Closer after "uploading".
+	// And we don't need to wrap the reader to anything else because the SDK will benefit from other interfaces like Seeker.
```

Comment:
Kindly remind @oliverpool @pat-s @viceice  that your fix is not proper.

Some SDK benefits from interfaces like "Seeker", so you can't just wrap it to a "NopCloser"

Reply:
But your fix changed `func (m *MinioStorage) Save(path string, r io.Reader, size int64) (int64, error)`, it affects all callers.

That's why it's improper.